intro-HPC/ch_running_batch_jobs.tex

```diff
@@ -40,6 +40,494 @@ \chapter{Running batch jobs}
+    Description:
+      This is just an example
+%\ifgent%
+    You will need to load all module(s) on any one of the lines below before the "example/1.2.3" module is available to load.
+
+      cluster/delcatty
+      cluster/golett
+      cluster/phanpy
+      cluster/swalot
+%\else%
+    This module can be loaded directly: module load example/1.2.3
+%\fi%
+    Help:
+
+      Description
+      ===========
+      This is just an example
+
+
+      More information
+      ================
+       - Homepage: https://example.com
+
+\end{prompt}
+
+\fi
+\subsection{Get detailed info}
+
+To get a list of all possible commands, type:
+
+\begin{prompt}
+%\shellcmd{module help}%
+\end{prompt}
+
+Or to get more information about one specific module package:
+
+\begin{prompt}
+%\shellcmd{module help example/1.2.3}%
+----------- Module Specific Help for 'example/1.2.3' ---------------------------
+  This is just an example - Homepage: https://example.com/
+\end{prompt}
+
+\ifusinglmod
+\subsection{Save and load collections of modules}
+If you have a set of modules that you need to load often,
+you can save these in a \emph{collection}. This will enable you to load all the
+modules you need with a single command.
+
+In each \lstinline|module| command shown below, you can replace \lstinline|module| with
+\lstinline|ml|.
+
+First, load all modules you want to include in the collections:
+
+\begin{prompt}
+%\shellcmd{module load \emph{example/1.2.3} \emph{secondexample/2.7-intel-2016b}}%
+\end{prompt}
+
+Now store it in a collection using \lstinline|module save|. In this example, the collection
+is named \lstinline|my-collection|.
+
+\begin{prompt}
+%\shellcmd{module save \emph{my-collection}}%
+\end{prompt}
+
+Later, for example in a jobscript or a new session, you can load all these modules with
+\lstinline|module restore|:
+
+\begin{prompt}
+%\shellcmd{module restore \emph{my-collection}}%
+\end{prompt}
+
+You can get a list of all your saved collections with the \lstinline|module savelist|
+command:
+
+\begin{prompt}
+%\shellcmd{module savelistr}%
+Named collection list (For LMOD_SYSTEM_NAME = "CO7-sandybridge"):
+  1) my-collection
+\end{prompt}
+
+To get a list of all modules a collection will load, you can use the \lstinline|module describe|
+command:
+\begin{prompt}
+%\shellcmd{module describe \emph{my-collection}}%
+1) example/1.2.3                                          6) imkl/11.3.3.210-iimpi-2016b
+2) GCCcore/5.4.0                                          7) intel/2016b
+3) icc/2016.3.210-GCC-5.4.0-2.26                          8) examplelib/1.2-intel-2016b
+4) ifort/2016.3.210-GCC-5.4.0-2.26                        9) secondexample/2.7-intel-2016b
+5) impi/5.1.3.181-iccifort-2016.3.210-GCC-5.4.0-2.26
+\end{prompt}
+\fi
+
+To remove a collection, remove the corresponding file in \lstinline|$HOME/.lmod.d|:
+
+\begin{prompt}
+%\shellcmd{rm \$HOME/.lmod.d/my-collection}%
+\end{prompt}
+
+\ifusinglmod
```

Comment:
`module show` is not specific to Lmod (`module spider` is though), so this shouldn't be guarded